modules/AlphaWalletFoundation/AlphaWalletFoundation/NFT/NFTProvider.swift

```diff
@@ -53,8 +59,13 @@ public final class AlphaWalletNFTProvider: NFTProvider {
     public func nonFungible() -> AnyPublisher<NonFungiblesTokens, Never> {
         let key = AddressAndRPCServer(address: wallet.address, server: server)
 
-        let tokensFromOpenSeaPromise = getOpenSeaNonFungible().publisher(queue: .global()).replaceError(with: [:])
-        let enjinTokensPromise = enjin.fetchTokens(wallet: wallet).receive(on: DispatchQueue.global()).mapToVoid().replaceError(with: ())
+        let tokensFromOpenSeaPromise = openSea.nonFungible(wallet: wallet)
+            .replaceError(with: [:])
+
+        let enjinTokensPromise = enjin.fetchTokens(wallet: wallet)
+            .receive(on: DispatchQueue.global())
```

Comment:
Ah, I see it used to be `.global` before this PR. Maybe we can switch this back to `.main`?